### ISSUE TITLE
FEATURE REQUEST: Support vertstat format?

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/RMINC#69]
The older mni.cortical.statistics tool could write out a file with a header, which brain-view, and other MNI vertstat* programs could understand.

It would be nice if RMINC could save in that format as well.

**Comment by cfhammill** [comment]
Can you give me a reference to the file format? I can investigate adding support for this

**Comment by gdevenyi** [comment]
Yeah sure, the ``mni.write.vertex.stats`` command in http://packages.bic.mni.mcgill.ca/tgz/mni_cortical_statistics-0.9.4.tar.gz implements saving in that format

This looks like the documentation of the format https://github.com/BIC-MNI/oobicpl/tree/master/docs (uncompiled)
